Show reviews

Egoes98 opened the pull request:
The methods and GUI for showing the reviews. When a guest searches for a hotel and enters the view where rooms are shown there appears a button that show a GUI dispalying the Reviews teh hotel has and also the average score.

Egoes98 committed:
Methods fro retrieving the Review

Egoes98 committed:
Added the view

Added a view to see the reviews. The view is not finished must be chnaged to look and function like with the view sistem.

Egoes98 committed:
Update ShowHotelReviewView.java

Egoes98 committed:
ShowHotelReviewView

I have addapted the view to the view system and added the localization and logic behind the reviews. It should be add a way of getting the hotelID and once that done it should work just fine. Also added to the overall schema.

Egoes98 committed:
Added View

Added the buttom to display the view. Added to the View factory. doesnt work yet because HotelID is not set.

Egoes98 committed:
Adde the button to panel

Egoes98 committed:
Update ShowHotelReviewView.java

The localization must be changed becaus is not displaying the text correctly. The view is show and working. The stetic must be improved. Also add the dispose.

Egoes98 committed:
Documentation-Localization

Added some documentation and localization. Some visual enancements must be done.

Egoes98 committed:
Some View improvements

Egoes98 committed:
View and Logic(Two erros)

The task should be finisehd but i get two errors that i dont know how to solve. The thing is that first time works fine but the seccond

Egoes98 committed:
Update ShowHotelReviewView.java

Egoes98 committed:
Merge branch 'master' into show-reviews

iapellaniz commented on src/test/java/es/deusto/spq/server/data/dto/AssemblerTest.java:
```diff
@@ -226,12 +226,10 @@ public void assembleReview_Review_Assemble() {
+                timestamp
         );
 
-        Assert.assertEquals(reviewDTO, assembler.assembleReview(review));
```
Why this change?

Egoes98 replied:
yeah sorry i will put it back as it was, it is becaus ei changed the dto and at first i thought i should change that also but then i realized that wasnt the case.

iapellaniz commented on src/main/java/es/deusto/spq/client/gui/views/reviews/ShowHotelReviewView.java:
```diff
@@ -0,0 +1,181 @@
+		lblAverageScore.setBounds(191, 47, 85, 25);
+		frame.getContentPane().add(lblAverageScore);
+		
+		frame.setVisible(true);
```
Missing `addDisposeEventHandler` method call at the end of initialization.

Egoes98 committed:
Minor Changes and Added isUnique

I have added the is Unique that i missed, but it causes error, i will try to solbe them.

Egoes98 committed:
Update ShowHotelReviewView.java

Egoes98 committed:
Bug fixed

It was giving error when you clicked various times the Show review, now it should be solved.

Egoes98 committed:
Finals removed

I removed some automated final eclipse set.

Egoes98 committed:
Merge branch 'master' into show-reviews

Egoes98 committed:
Merge branch 'master' into show-reviews

iker-barriocanal reviewed (dismissed):
Would be useful to have an example review to see it.

Egoes98 committed:
Merge branch 'master' into show-reviews

Egoes98 committed:
Example Data

Egoes98 commented:
I have added a file with the SQL needed to create the example data. You have to create the DB and then execute this SQL statements.

Egoes98 committed:
Merge branch 'master' into show-reviews

iapellaniz commented:
Tests are failing due to performance issues. Maybe we should increase the limit to 300ms

Egoes98 commented:
> Tests are failing due to performance issues. Maybe we should increase the limit to 300ms

Could be a good idea, i will change it.

Egoes98 committed:
Changed RMI test time

From 200ms to 300ms

Egoes98 committed:
Merge branch 'master' into show-reviews

Egoes98 committed:
Merge branch 'master' into show-reviews
